pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -771,6 +774,11 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(logger klog.Logger, pInfo *
 		return fmt.Errorf("Pod %v is already present in the backoff queue", klog.KObj(pod))
 	}
 
+	if pInfo.UnschedulablePlugins.Union(pInfo.PendingPlugins).Len() != 0 {
+		// This Pod is rejected by some plugins, not coming back due to unexpected errors (e.g., a network issue)
```

Comment:
From the description in https://github.com/kubernetes/kubernetes/issues/128744

```
* Unschedulable: the Pod is unschedulable; e.g., no node has enough resource, nodeaffinity, etc.
* Error: the Pod cannot be scheduled because of unexpected errors; e.g., kube-apiserver is unstable, network issue, etc.
```

UnschedulableCount should not increase when `a network issue`?

Reply:
Should not, and here describes/implements that, right?

Reply:
OK, the comment "rejected by some plugins" here makes me think some plugins refer to  UnschedulablePlugins/PendingPlugins.